### PR TITLE
Readme tweaks

### PR DESCRIPTION
**Pull request by ewels**
Suggested tweaks to make the readme easier to read and a more informative for newcomers..

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `README.md` is updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,25 +3,32 @@
...
+
+The pipeline supports short-read illumina data from both shotgun and enriched library preparation methods (amplicon, target enrichment).
 
-This pipeline is a re-implementation of the [SARS_Cov2_consensus-nf](https://github.com/BU-ISCIII/SARS_Cov2_consensus-nf) and [SARS_Cov2_assembly-nf](https://github.com/BU-ISCIII/SARS_Cov2_assembly-nf) pipelines initially developed by [Sarai Varona](https://github.com/svarona) and [Sara Monzon](https://github.com/saramonzon) from [BU-ISCIII](https://github.com/BU-ISCIII). Porting both of these pipelines to nf-core was an international collaboration between numerous contributors and developers, led by [Harshil Patel](https://github.com/drpatelh) from the [The Bioinformatics & Biostatistics Group](https://www.crick.ac.uk/research/science-technology-platforms/bioinformatics-and-biostatistics/) at [The Francis Crick Institute](https://www.crick.ac.uk/), London. We appreciated the need to have a portable, reproducible and scalable pipeline for the analysis of COVID-19 sequencing samples and so the Avengers Assembled! Please come and join us and add yourself to the contributor list :)
```
Comment (ewels) [inline README.md:15]: Whilst this paragraph is nice, it seems to be essentially a duplicate of the information that is contained in the _Credits_ section below. So I deleted it. Let me know if you'd like to keep any of this information (preferably moved down to credits).

File: README.md [hunk 2]
```diff
@@ -3,25 +3,32 @@
 [![GitHub Actions CI Status](https://github.com/nf-core/viralrecon/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/viralrecon/actions)
 [![GitHub Actions Linting Status](https://github.com/nf-core/viralrecon/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/viralrecon/actions)
 [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A519.10.0-brightgreen.svg)](https://www.nextflow.io/)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3872730.svg)](https://doi.org/10.5281/zenodo.3872730)
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/nfcore/viralrecon.svg)](https://hub.docker.com/r/nfcore/viralrecon)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3872730.svg)](https://doi.org/10.5281/zenodo.3872730)
+[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23viralrecon-4A154B?logo=slack)](https://nfcore.slack.com/channels/viralrecon)
```
Comment (drpatelh) [inline README.md:10]: 💯

File: README.md [hunk 3]
```diff
@@ -49,25 +56,23 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
 ## Quick Start
 
-i. Install [`nextflow`](https://nf-co.re/usage/installation)
-
-ii. Install either [`Docker`](https://docs.docker.com/engine/installation/) or [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) for full pipeline reproducibility (please only use [`Conda`](https://conda.io/miniconda.html) as a last resort; see [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles))
-
-iii. Download the pipeline and test it on a minimal dataset with a single command
+1. Install [`nextflow`](https://nf-co.re/usage/installation)
```
Comment (drpatelh) [inline README.md:59]: Will need to change some of this stuff in the template too 😅

File: README.md [hunk 4]
```diff
@@ -3,25 +3,32 @@
...
 
-We have integrated a number of options in the pipeline to allow you to run specific aspects of the workflow if you so wish. For example, you can skip all of the assembly steps with the `--skip_assembly` parameter. See [usage docs](docs/usage.md) for all of the available options when running the pipeline.
+## Example output
 
-Please click [here](https://raw.githack.com/nf-core/viralrecon/master/docs/html/multiqc_report.html) to see an example MultiQC report generated using the parameters defined in [this configuration file](https://github.com/nf-core/viralrecon/blob/master/conf/test_full.config) to run the pipeline on [samples](https://zenodo.org/record/3735111) which were prepared from the [ncov-2019 ARTIC Network V1 amplicon set](https://artic.network/ncov-2019) and sequenced on the Illumina MiSeq platform in 301bp paired-end format.
+The viralrecon pipeline provides analysis on a single target genome of interest, such as a virus. Numerous QC and reporting steps are included, finally finishing with a single [MultiQC](https://multiqc.info/) report containing a full summary of the analysis.
```
Comment (drpatelh) [inline README.md:22]:
```suggestion
Numerous QC and reporting steps are included in the pipeline, and collated within a single [MultiQC](https://multiqc.info/) report containing a full summary of the analysis. You can see [an example MultiQC report here](https://raw.githack.com/nf-core/viralrecon/master/docs/html/multiqc_report.html), generated using the parameters defined in [this configuration file](https://github.com/nf-core/viralrecon/blob/master/conf/test_full.config). The pipeline was run with [these samples](https://zenodo.org/record/3735111), prepared from the [ncov-2019 ARTIC Network V1 amplicon set](https://artic.network/ncov-2019) and sequenced on the Illumina MiSeq platform in 301bp paired-end format. 
```

File: README.md [hunk 5]
```diff
@@ -3,25 +3,32 @@
...
 
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible. Furthermore, automated continuous integration tests to run the pipeline on a full-sized dataset are passing on AWS cloud.
+To get a feel for what to expect, you can see [an example MultiQC report here](https://raw.githack.com/nf-core/viralrecon/master/docs/html/multiqc_report.html), generated using the parameters defined in [this configuration file](https://github.com/nf-core/viralrecon/blob/master/conf/test_full.config). The pipeline was run with [these samples](https://zenodo.org/record/3735111), prepared from the [ncov-2019 ARTIC Network V1 amplicon set](https://artic.network/ncov-2019) and sequenced on the Illumina MiSeq platform in 301bp paired-end format.
```
Comment (drpatelh) [inline README.md:24]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -3,25 +3,32 @@
...
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible. Furthermore, automated continuous integration tests that run the pipeline on a full-sized dataset using AWS cloud ensure that the code is stable.
 
-We have integrated a number of options in the pipeline to allow you to run specific aspects of the workflow if you so wish. For example, you can skip all of the assembly steps with the `--skip_assembly` parameter. See [usage docs](docs/usage.md) for all of the available options when running the pipeline.
+## Example output
```
Comment (drpatelh) [inline README.md:20]:
```suggestion
## Pipeline reporting
```

File: README.md [hunk 7]
```diff
@@ -3,25 +3,32 @@
...
 [![Docker](https://img.shields.io/docker/automated/nfcore/viralrecon.svg)](https://hub.docker.com/r/nfcore/viralrecon)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3872730.svg)](https://doi.org/10.5281/zenodo.3872730)
+[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23viralrecon-4A154B?logo=slack)](https://nfcore.slack.com/channels/viralrecon)
 
 ## Introduction
 
-**nfcore/viralrecon** is a bioinformatics analysis pipeline used to perform assembly and intrahost/low-frequency variant calling for viral samples. The pipeline currently supports metagenomics and amplicon sequencing data derived from the Illumina sequencing platform.
+**nfcore/viralrecon** is a bioinformatics analysis pipeline used to perform assembly and intrahost/low-frequency variant calling for viral samples.
+
+The pipeline supports short-read illumina data from both shotgun and enriched library preparation methods (amplicon, target enrichment).
```
[the remark this answers is not in the document]
Comment (saramonzon) [reply README.md:16]:
```suggestion
The pipeline supports short-read Illumina sequencing data from both shotgun (e.g. sequencing directly from clinical samples) and enrichment-based library preparation methods (e.g. amplicon-based: [ARTIC SARS-CoV-2 enrichment protocol](https://artic.network/ncov-2019); or probe-capture-based).
```

File: README.md (hunk 3 above)
Comment (ewels) [reply README.md:59]: Yeah, I wondered that.. 🙄

Comment (ewels) [reply README.md:57]: https://github.com/nf-core/tools/pull/621

**Comment by ewels** [comment]
Comments applied - ready for review & merge now I think 👍 Thanks both!

**Comment by drpatelh** [comment]
Thanks alot @ewels 🙂

**Pull request merged**